applications/rag/workloads.tfvars

```diff
@@ -12,16 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-project_id = "<your project ID>"
+project_id      = "<your project ID>"
+create_network  = true         # Creates a new VPC for your cluster. Disable to use an existing network.
+network_name    = "ml-network" # Creates a network named ml-network by default. If using an existing VPC, ensure you follow the README instructions to enable Private Service Connect for your VPC.
+subnetwork_cidr = "10.100.0.0/16"
 
-## this is required for terraform to connect to GKE master and deploy workloads
-create_cluster    = true # Create a GKE cluster in the specified network.
+create_cluster    = true # Creates a GKE cluster in the specified network.
 autopilot_cluster = true
 cluster_name      = "<cluster_name>"
```

Comment:
Can we put this at the top alongside `project_id`? It's nice to group the required user-defined fields as much as possible.

Reply:
I personally prefer to leave them as in, it's grouped by network config, cluster config, etc..
Let's redo this in a future PR